config/locales/client.en.yml

```diff
@@ -4877,6 +4878,7 @@ en:
             address_placeholder: "name@example.com"
             type_placeholder: "digest, signup..."
             reply_key_placeholder: "reply key"
+            smtp_transaction_response_placeholder: "smtp ID"
```

Comment:
I'm wondering why `smpt` is lowercase here while uppercase elsewhere.

Reply:
I probably just got influenced by the other placeholders :) Will fix